### ISSUE TITLE
2023.8.7 Update

### ISSUE THREAD

**Issue by xkuzma** [cursey/kanan-new#300]
Errors keep occurring
-Failed to find address of CRenderer 
-Failed to find CWorld
-Failed to find CAccount

**Comment by Vitashique** [comment]
Any news on this?

**Comment by xkuzma** [comment]
> Any news on this?

None that I know of.

**Comment by shaggyze** [comment]
> Any news on this?

yes, there was a pull request #299 10 hours ago that Cursey merged and released 2 hours ago..

**Comment by Vitashique** [comment]
Errors are resolved. Now it gives error: "Another Mabinogi client is already running" when trying to run multi-client. I used both Hyddwn Launcher and Kanan Launcher. Error appeared for both launchers.

**Comment by Shulzs** [comment]
after 2023.8.10 update : 
-Failed to find CEntityList.

**Comment by dyelikeaboss** [comment]
After merging the pull request, kanan injects but equipment override is not working.

[EquipmentOverride] Entering constructor
[EquipmentOverride] Failed to find address of setEquipmentInfo!
[EquipmentOverride] Leaving constructor

**Comment by kohupallintrax** [comment]
Last patch was a pretty major one - most patches will be broken. Pull request  #299  was a quick fix to get Kanan in a state where it could be injected without crashing. The patches I've authored and maintained will likely be fixed by the end of the weekend. 

These kinds of updates that break everything happen once every other year or so.
